components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -3585,10 +3444,10 @@ public static NativeArray jsFunction_getProviderAPIVersionUserUsage(Context cx,
         String apiName = (String) args[1];
         String version = (String) args[2];
         try {
-            APIUsageStatisticsClient client = new APIUsageStatisticsClient(((APIProviderHostObject) thisObj).getUsername());
+            APIUsageStatisticsRdbmsClientImpl client = new APIUsageStatisticsRdbmsClientImpl(((APIProviderHostObject) thisObj).getUsername());
```

Comment:
already removed methods from hostobjects in commit https://github.com/wso2/carbon-apimgt/commit/fd35737f6a9fb080e69b6e915d9ed0f1eee2b6df